docs/source/quickstart-docker.rst

```diff
@@ -1,10 +1,16 @@
 Install JupyterHub with Docker
 ==============================
 
+
 The JupyterHub `docker image <https://hub.docker.com/r/jupyterhub/jupyterhub/>`_ is the fastest way to set up Jupyterhub in your local development environment.
 
-The ``JupyterHub`` docker image runs the Hub service only. It does not provide other Jupyter components such as Notebook installation, which are needed by the single-user servers.
-To run the single-user servers, which may be on the same system as the Hub or not, Jupyter Notebook or Jupyter Lab must be installed.
+.. note::
+    This ``jupyterhub/jupyterhub`` docker image is only an image for running
+    the Hub service itself. It does not provide the other Jupyter components,
+    such as Notebook installation, which are needed by the single-user servers.
+    To run the single-user servers, which may be on the same system as the Hub or
+    not, `Jupyter Notebook <https://jupyterlab.readthedocs.io/>`_ or greater must be installed.
```

Comment:
`or greater` here doesn't work - or greater would follow a version number. Plus, the text is "Jupyter Notebook" but the link is for JupyterLab. Let's say: "JupyterLab (link) or Jupyter Notebook (no link) must be installed"

Reply:
> `or greater` here doesn't work - or greater would follow a version number. Plus, the text is "Jupyter Notebook" but the link is for JupyterLab. Let's say: "JupyterLab (link) or Jupyter Notebook (no link) must be installed"

ok sir @minrk thanks